frontend/src/main/pages/ProfilePage.js

```diff
@@ -28,13 +86,51 @@ const ProfilePage = () => {
                 <Col md>
                     <h2>{fullName}</h2>
                     <p className="lead text-muted">{email}</p>
+        
+                    
                     <RoleBadge role={"ROLE_USER"} currentUser={currentUser}/>
                     <RoleBadge role={"ROLE_MEMBER"} currentUser={currentUser}/>
                     <RoleBadge role={"ROLE_ADMIN"} currentUser={currentUser}/>
                     <RoleBadge role={"ROLE_DRIVER"} currentUser={currentUser}/>
                     <RoleBadge role={"ROLE_RIDER"} currentUser={currentUser}/>
+                    <p></p>
+                    <>
+                    <p className="lead text-muted" style={{ marginRight: "8px"}}>{"cell phone number: "} {whichNumber ? (updatedPhoneNumber ? updatedPhoneNumber : "N/A") : (cellPhone ? cellPhone : "N/A")}</p>
+      <Button variant="primary" onClick={handleShow}>
+        Change Cell Phone Number
+      </Button>
 
+      <Modal show={show} onHide={handleClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Input phone number</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Form onSubmit={onSubmit}>
+            <Form.Group className="mb-3">
+              <Form.Label>Phone number</Form.Label>
+              <Form.Control
+                type="text"
+                id="PhoneInput"
+                data-testid="PhoneInput"
+                placeholder="###-###-####"
+                autoFocus
+                onChange={onChangePhoneNumber} // Update phone number state
```

Comment:
```suggestion
                onChange={onChangePhoneNumber}
```

I wonder whether the comment is adding any valuable information here, or is just clutter?